app/src/main/res/values-ko/strings.xml

```diff
@@ -1,279 +1,338 @@
-    <string name="comment_node_more_replies">%1$s 개의 추가 답글</string>
-    <string name="comment_node_view_post">게시물 보기</string>
-    <string name="comment_node_view_context">문맥 보기</string>
-    <string name="comment_body_removed">*제거됨*</string>
+    <string name="AppBars_comments">%1$s 개의 댓글</string>
+    <string name="AppBars_created">Created</string>
+    <string name="AppBars_posts">%1$s 개의 게시물</string>
+    <string name="AppBars_users_6_months">%1$s 사용자 / 6 개월</string>
+    <string name="AppBars_users_day">%1$s 사용자 / 일</string>
+    <string name="AppBars_users_month">%1$s 사용자 / 월</string>
+    <string name="AppBars_users_week">%1$s 사용자 / 주</string>
+    <string name="about_social">소셜</string>
+    <string name="account_settings_active">활발</string>
+    <string name="account_settings_activity_account_settings">계정 설정</string>
+    <string name="account_settings_all">전체</string>
+    <string name="account_settings_avatar">아바타</string>
+    <string name="account_settings_banner">배너</string>
+    <string name="account_settings_bio">자기소개</string>
+    <string name="account_settings_bot_account">봇 계정</string>
+    <string name="account_settings_default_listing_type">기본 표시 목록</string>
+    <string name="account_settings_default_sort_type">기본 정렬 기준</string>
+    <string name="account_settings_display_name">표시 이름</string>
+    <string name="account_settings_email">이메일</string>
+    <string name="account_settings_hot">인기</string>
+    <string name="account_settings_local">로컬</string>
+    <string name="account_settings_matrix_user">Matrix 사용자</string>
+    <string name="account_settings_mostcomments">최다 댓글</string>
+    <string name="account_settings_new">신규</string>
+    <string name="account_settings_newcomments">신규 댓글</string>
+    <string name="account_settings_old">오래됨</string>
+    <string name="account_settings_remove_current_avatar">현재 아바타 삭제</string>
+    <string name="account_settings_save_settings">설정 저장</string>
+    <string name="account_settings_send_notifications_to_email">알림을 이메일로 보내기</string>
+    <string name="account_settings_show_avatars">아바타 표시</string>
+    <string name="account_settings_show_bot_accounts">봇 계정 표시</string>
+    <string name="account_settings_show_notifications_for_new_posts">새 게시물 알림 표시</string>
+    <string name="account_settings_show_nsfw">후방주의 게시물 표시</string>
+    <string name="account_settings_show_read_posts">읽은 게시물 표시</string>
+    <string name="account_settings_show_scores">점수 표시</string>
+    <string name="account_settings_subscribed">구독</string>
+    <string name="account_settings_topall">전체 최고</string>
+    <string name="account_settings_topday">일 최고</string>
+    <string name="account_settings_topmonth">월 최고</string>
+    <string name="account_settings_topweek">주 최고</string>
+    <string name="account_settings_topyear">연 최고</string>
+    <string name="addBookmark">북마크에 추가</string>
+    <string name="app_bars_dot_spacer">·</string>
+    <string name="app_name">Jerboa for Lemmy</string>
```

Comment:
This line is no longer needed as `app_name` is now set to `translatable=false`

See https://github.com/dessalines/jerboa/pull/904